### DISCUSSION TITLE
类型“IntrinsicAttributes & IProviderProps”上不存在属性“children”

### DISCUSSION THREAD

**Discussion by huangshiqiang** [alibaba/formily#3057] [Q&A 2.x]
使用formily组件，很多TS错误，请问需要如何解决这个问题？以下代码是官网中的事例，我觉得应该在相关的类型加上children。
<img width="855" alt="image" src="https://user-images.githubusercontent.com/6261601/164353498-448fcf88-d6ea-4937-aa59-c56a60e986b2.png">

**Comment by dohard-ma** [comment]
JSON Schema 的写法，可以规避这个问题。例如 https://github.com/formilyjs/antd-mobile/blob/master/docs/components/FormLayout.zh-CN.md#json-schema-%E6%A1%88%E4%BE%8B
